ledger/src/genesis_ledger.rs

```diff
@@ -32,6 +45,15 @@ where
         self.into_iter()
     }
 
+    /// Build mina merkle ledger tree with a fixed height
+    fn to_mina_merkle_ledger(&'a self) -> MinaLedgerMerkleTree {
+        const MINA_LEDGER_HEIGHT: u32 = 20;
```

Comment:
adding a comment here mentioning why we are using `MINA_LEDGER_HEIGHT = 20`, would be good for future reference

Reply:
Comment added